chia/clvm/spend_sim.py

```diff
@@ -253,6 +253,7 @@ async def farm_block(
         generator_bundle: Optional[SpendBundle] = None
         return_additions: List[Coin] = []
         return_removals: List[Coin] = []
+        spent_coins_ids = None
```

Comment:
this line seems unnecessary. Is there any scenario where you access this variable before assigning a value to it?

Reply:
An example code path is when `peak` is `None`.

Reply:
I see